support-frontend/assets/helpers/__tests__/urlTest.ts

```diff
@@ -1,12 +1,18 @@
 // ----- Imports ----- //
+
+import type { JSDOM } from 'jsdom';
 import {
 	addQueryParamsToURL,
 	getAbsoluteURL,
 	getAllQueryParams,
 	getAllQueryParamsWithExclusions,
 } from '../urls/url';
+
 // ----- Tests ----- //
-const { jsdom } = global;
+
+// @ts-expect-error -- This is added to the global scope by the test setup
+const jsdom = global.jsdom as JSDOM;
```

Comment:
I think it's a bit of a smell that so many of our tests directly interact with jsdom. Really here there should probably be a couple of methods to get the query string and set the query string which interact with the dom, every thing else (i.e parsing and manipulating) could just be pure functions that are much easier to test

Reply:
(everything below this change is just adding spaces)